components/dialog/RotateEncryptionKeyDialog.vue

```diff
@@ -1,13 +1,15 @@
 <script>
+import { SNAPSHOT, NORMAN } from '@/config/types';
 import AsyncButton from '@/components/AsyncButton';
 import Card from '@/components/Card';
 import Banner from '@/components/Banner';
 import { exceptionToErrorsArray } from '@/utils/error';
-import { NORMAN } from '@/config/types';
+
 import { sortBy } from '@/utils/sort';
 import day from 'dayjs';
 import { escapeHtml } from '@/utils/string';
 import { DATE_FORMAT, TIME_FORMAT } from '@/store/prefs';
+import _ from 'lodash';
```

Comment:
Be careful here you are importing the whole Lodash.

The import should be done for each function.

Reply:
changed to `import { set } from 'lodash';`